docs/platforms.md

```diff
@@ -0,0 +1,41 @@
+# Platforms
+
+This document outlines the supported platforms for matrix.org bridge projects.
+
+### Bridge versioning
+
+All bridge application and library projects aim to follow the [Semantic Versioning](https://semver.org/) system.
+
+### Node version support
+
+Bridge projects will always support the **Active LTS** release and the **Current** release of Node.JS. When a
+Node.JS version falls from Active to Maintenance, we will migrate projects to the next Active release over time.
+
+At time of writing (July 2022) we support 16.X and 18.X as they are the Active and Current releases of Node.
+
+Bridge projects do not support odd-versioned Node.JS releases, as these are short lived non-LTS versions and are
+difficult to support since they have a 6 month shelf life.
+
+See https://nodejs.org/en/about/releases/ for more information about Node.JS releases.
+
+### LTS releases
+
+We do not currently have a LTS process for bridge projects. The latest released version of the bridge is
+what we aim to support. Versions a few points behind latest might be eligible for some support, but the
+advice is always to upgrade as soon as possible.
+
+### Platforms
+
+In terms of installation platforms, we support as a baseline:
+
+- Installation onto a host directly (via git) on Windows, Mac, Linux.
+- Docker (x86_64) on Windows, Mac, Linux.
+
+Support for other architectures is something we aim to do, for instance ARM but at the moment
+this is best-effort.
+
+In a few cases, bridges also have a maintained version on npm but this is not universally true.
```

Comment:
Do they have unmaintained version on npm, or no versions at all? If it's the former then we should probably drop them, before we forget to update them with security fixes or something :)

Reply:
I believe we always update the IRC bridge, but we don't provide it for the others.

Reply:
Decided to drop this statement in the end, given we only do it for IRC.